gulp/config.js

```diff
@@ -83,7 +69,57 @@ module.exports = (function() {
     var options = {
       files: [].concat(
         './node_modules/phantomjs-polyfill/bind-polyfill.js',
-        wiredep({devDependencies: true}).js,
+        './node_modules/jquery/dist/jquery.js',
```

Comment:
wiredep is for bower dependencies only ever just bower dependencies, i TOTALLY feel what you are feeling, wait what, more than one place to update deps?  I kinda ran outta time and ideas on how to getting those in here, open to suggestions!  

ultimately i was going to write up some documentation on how to correctly add dependencies and keep in mind, this is only until we move to angular 2!  importing modules will change a great deal of what we presently do...